path/src/cmd/protocolhandle.ps1

```
@@ -0,0 +1,29 @@
+	}
+	if (-not $protocolUrl) {
+		Write-Error (Get-I18n -key 'protocol.noUrl')
+		exit 1
+	}
+	# 编码 URL 参数，防止特殊字符问题，确保传入的 URL 能正确作为查询参数
+	$encodedUrl = [uri]::EscapeDataString($protocolUrl)
+	$targetUrl = "https://steve02081504.github.io/fount/protocol/?url=$encodedUrl"
+
+	Test-PWSHModule fount-pwsh
+	Start-Job -ScriptBlock {
+		param ($targetUrl)
```

PSScriptAnalyzer (1.25.0)

[warning] The variable '$targetUrl' is not declared within this ScriptBlock, and is missing the 'Using:' scope modifier.

Suggested fix: Replace $targetUrl with $using:targetUrl

(PSUseUsingScopeModifierInNewRunspaces)